docs/app/Index.rst

```diff
@@ -21,18 +22,17 @@ The web application offers the following features:
   * Current and historical device data
     
     * GNSS location
+    * approximate location using :ref:`nRF Cloud Location Services <app-cellgeolocation>` and other third-party providers
     * Battery voltage
-    * Accelerometer
+    * Temperature
+    * ... and more
```

Comment:
Not so sure about this last bullet. Maybe instead, add to the end of sentence on line 22 "such as" or "for example"